docs/point_api_spec.md

```diff
@@ -0,0 +1,69 @@
+# Address Point API Spec
+
+## Purpose
+This document describes the Address Point Geocoding API, serving as a specification or "contract" for other users of this service.
+In particular, it describes the endpoints that this service exposes, as well as the structure and format of the data that is exchanged when using this service.
+
+### Protocols and data formats
+The Address Point Geocoding Service exposes data through HTTP, accepting and returing data in JSON format. 
+In particular, all requests need to specify their content type as appropriate:
+
+```
+Content-Type: application/json
+```
+
+Requests that don't specify their content type will be rejected
+
+### API Endpoints
+
+`GET  /status`
+
+Returns a message with current status and date
+
+```json
+{
+    "status": "OK",
+    "time": "Wed Feb 18 10:30:55 EST 2015"
+}
+```
+
+`POST /address/point`
+
+Payload: JSON object with the following structure:
+
+* id: Identifier for the record (Integer)
+* address: Address to geocode (String)
+
+Example:
+
+```json
+{
+  "id": 1,
```

Comment:
Actually, if the address we are returning is the one Elasticsearch resolves from the query, an id seems more useful, since the returned id might not (or probably won't) match the query precisely. Still, seems like this should be the duty of client code to manage its own callbacks (or the JS API could populate a request callback argument with the original query, etc, rather than having to pass around an id)